js/renovation/ui/scroll_view/scrollable_native.tsx

```diff
@@ -310,4 +322,9 @@ export class ScrollableNative extends JSXComponent<ScrollableInternalPropsType>(
     };
     return combineClasses(classesMap);
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  get refreshStrategy(): RefreshStrategy | undefined {
+    return devices.real().platform === 'android' ? 'swipeDown' : undefined;
```

Comment:
We can move this code into the TopPocket component as a default value for refreshStrategy

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/15914/commits/8a0b34850957ec17aa4046544e90f522a3585075